nvflare/app_opt/xgboost/histogram_based_v2/adaptor_controller.py

```diff
@@ -134,6 +135,7 @@ def __init__(
         max_client_op_interval: float = Constant.MAX_CLIENT_OP_INTERVAL,
         progress_timeout: float = Constant.WORKFLOW_PROGRESS_TIMEOUT,
         client_ranks=None,
+        enable_reliable_sender=True,
```

Comment:
We can always enable reliable sender, so no need to config here.
It won't have any effect if the client side doesn't use reliable sender, but this is okay.

Reply:
It's not really needed. Removed.